cms/views.py

```diff
@@ -348,3 +353,45 @@ def change_password(request):
     context['form'] = form
     context.update(csrf(request))
     return render(request, 'cms/templates/change_password.html', context)
+    
+def confirm_student(request, mdlid):
+    user_hashid = Hashids(salt = settings.SPOKEN_HASH_SALT)
+    mdluserid = user_hashid.decode(mdlid)
+    try:
+        mdluser = MdlUser.objects.filter(pk=mdluserid[0]).first()
+        user = User.objects.get(email=mdluser.email)
+        student = Student.objects.get(user_id = user.id)
+        if mdluser:
+            user.is_active = True
+            user.save()
+            
+            student.verified = True
+            student.error = False
+            student.save()
+            
+            messages.success(request, "Your account has been activated!. Please login to continue.")
+            return HttpResponseRedirect('http://spoken-tutorial.org/participant/login/')
+        else:
+            print 'can not match record'
+            messages.error(request, "Your account not activated!. Please try again!")
+            return HttpResponseRedirect('/')
+    except Exception, e:
```

Comment:
`except Exception` needs to replace with proper Exception class, maybe `ObjectNotExist`